subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c

```diff
@@ -90,10 +90,14 @@ static void rtc0_nrf5_isr(void *arg)
 
 	mayfly_run(TICKER_USER_ID_ULL_HIGH);
 
+#if (CONFIG_BT_CTLR_ULL_HIGH_PRIO == CONFIG_BT_CTLR_ULL_LOW_PRIO)
+	mayfly_run(TICKER_USER_ID_ULL_LOW);
```

Comment:
@cvinayak this worries me a bit: I want to avoid that the ISR (i.e. mayfly_run) is called twice for the same event...

Reply:
Oh, now, I see it, below: the ISR is not wired if the priorities are the same...